test/integration/scripts.test.js

```diff
@@ -80,7 +80,7 @@ describe("Command-Line Scripts", () => {
             }
             const restore = spawnSync('node', args);
             let expectedContent = fs.readFileSync(expected, 'utf8')
-            if (extension === extensions.text) {
+            if (extension !== extensions.markdown) {
```

Comment:
is there a specific reason for this switch?

Reply:
I was having problems with added newlines to the expected content, despite there not being any in the files.